modules/andes-core/broker/src/main/java/org/wso2/andes/thrift/ThriftClientFactory.java

```diff
@@ -111,7 +111,7 @@ public boolean validateObject(PooledObject<SlotManagementService.Client> client)
         try {
             return client.getObject().healthCheck();
         } catch (TException e) {
-            log.error("exception occurs while validating the thrift connection" + e);
+            log.debug("exception occurs while validating the thrift connection" + e);
```

Comment:
parse the e as argument rather than appending to message

Reply:
seems the error message is misleading because it says exception occurs while validating. Lets change it to "Connection is not active. Hence removing the connection from pool"